cimr/prompt.py

```diff
@@ -271,8 +289,8 @@ def main():
             raise ValueError(' invalid log level: %s' % loglevel)
         logging.basicConfig(level=numeric_level)
     else:
-        logging.error(f' --log argument must be debug, info, warning, error, or critical.')
-        logging.error(f' --log level is set to \'info\' by default.')
+        logging.error(f' -log argument must be debug, info, warning, error, or critical.')
+        logging.error(f' -log level is set to \'info\' by default.')
```

Comment:
Probably better to use `argument` instead of `level`, since you are using the former in line 292.